src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/RegenerateConditionalOrderPronouncedCoverLetter.java

```diff
@@ -25,54 +25,60 @@ public CaseDetails<CaseData, State> apply(CaseDetails<CaseData, State> caseDetai
 
         final Long caseId = caseDetails.getId();
         final CaseData caseData = caseDetails.getData();
-
         if (caseData.getApplicant1().isApplicantOffline() && caseData.getApplicant1().isConfidentialContactDetails()) {
+            removeAndRegenerateApplicant1(caseId, caseData, coverLetterHelper);
+        }
+        if (caseData.getApplicant1().isApplicantOffline() && caseData.getApplicant2().isConfidentialContactDetails()) {
```

Comment:
Should it be applicant 2 here?
```suggestion
        if (caseData.getApplicant2().isApplicantOffline() && caseData.getApplicant2().isConfidentialContactDetails()) {
```